### ISSUE TITLE
Documentation: --verbosity parameter not documented in README

### ISSUE THREAD

**Issue by Eorhim** [GoogleContainerTools/kaniko#638]
**Actual behavior**
--verbosity is listed in the short usage help, if you run kaniko without any parameters. It is not listed in README.md

**Expected behavior**
It should be listed in both places.

**To Reproduce**
Compare the output of

`$ docker run --rm gcr.io/kaniko-project/executor:latest`

with the list of additional flags in README.md.

**Additional Information**
  - Kaniko Image (fully qualified with digest)
    gcr.io/kaniko-project/executor@sha256:d9fe474f80b73808dc12b54f45f5fc90f7856d9fc699d4a5e79d968a1aef1a72

**Comment by Rakaim** [comment]
I too was searching the README for this option! Thank you for informing me that it's in the output of Kaniko!

I'll put this here for other lost souls like me:
```
-v, --verbosity string                          Log level (debug, info, warn, error, fatal, panic (default "info")
```

**Comment by sharifelgamal** [comment]
This is fixed with #634
